### PR TITLE
implement physics for player, add ground entity

### PR DESCRIPTION
**Pull request by cdsupina**
- Add bevy raper_2d for physics
- Changed player movement to use rapier physics
- Added gravity
- Created ground
- Gave player and ground rigidbodies and colliders
- Added optimization levels for dev profile

### CODE REVIEW

File: src/runner/backgroundlayer.rs [hunk 1]
```diff
@@ -9,7 +9,7 @@ pub struct BackgroundLayerPlugin;
 impl Plugin for BackgroundLayerPlugin {
     fn build(&self, app: &mut App) {
         app.add_startup_system(spawn_background)
-            .add_system(move_background);
+            .add_system(move_background.system());
```
Comment (tigleym) [inline src/runner/backgroundlayer.rs:12]: It looks like this was added back? I think we can get rid of `system()` here

**Review by tigleym — APPROVED** [review]
This looks good to me! I had question about the `system()` call, but other than that I this we can get this in 👍

File: src/runner/backgroundlayer.rs (hunk 1 above)
Comment (NightlySide) [reply src/runner/backgroundlayer.rs:12]: Yes, it's better to remove the `.system()` suffix, maybe it is a leftover from the 0.5 experience and @cdsupina's tiredness

**Review by NightlySide — CHANGES_REQUESTED** [review]

File: src/platforms/mod.rs [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+}
+
+/// Spawn ground under player
+fn spawn_ground(
+    mut commands: Commands,
+    rapier_config: Res<RapierConfiguration>,
+    asset_server: Res<AssetServer>,
+) {
+    let collider_size_hx = 2000.0 / rapier_config.scale / 2.0;
+    let collider_size_hy = 24.0 / rapier_config.scale / 2.0;
```
Comment (tigleym) [inline src/platforms/mod.rs:18]: This seems to be a fixed size.  Should we eventually make it so this extends off screen?

**Pull request merged**